R/init.R

```diff
@@ -100,9 +100,99 @@ init_last <- function() {
     rm(".First.sys", envir = globalenv())
   )
 
+
   # Attach to vscode
   exports$.vsc.attach()
 
+  # Import VSC-R settings
```

Comment:
Some settings (e.g. `vsc.plot`, `vsv.use_httpgd`, `vsc.rstudioapi`) must be set before the `init.R` is sourced.